tests/metagen/typegraphs/identities/py/fdk.py

```
@@ -0,0 +1,1920 @@
+        ty_to_gql_ty_map: typing.Dict[str, str],
+    ):
+        self.gql_fn = gql_fn
+        self.opts = opts
+        self.ty_to_gql_ty_map = ty_to_gql_ty_map
+
+    def fetch(
+        self,
+        doc: str,
+        variables: typing.Dict[str, typing.Any],
+        opts: typing.Optional[GraphQLTransportOptions],
+        files: typing.Dict[str, File] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)